src/Microsoft.Azure.WebJobs.ServiceBus/EventHubs/IEventHubProvider.cs

```diff
@@ -8,6 +8,9 @@ namespace Microsoft.Azure.WebJobs.ServiceBus
     // Expose to binders / attributes so they can get the EventHub connections. 
     internal interface IEventHubProvider
     {
+        // Gets the checkpointing frequency
```

Comment:
Are you rebased on dev? I don't even see this IEventHubProvider type existing anymore